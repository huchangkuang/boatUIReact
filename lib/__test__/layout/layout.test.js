import 'jsdom-global/register';
import renderer from "react-test-renderer";
import Adapter from 'enzyme-adapter-react-16';
import {configure} from "enzyme";
import Layout from "../../layout/layout";
import Header from "../../layout/header";
import Content from "../../layout/content";
import Footer from "../../layout/footer";
import React from "react";

configure({adapter: new Adapter()});
describe('layout', () => {
  it('should be div', function () {
    const component = <Layout style={{height: 500, width: 400}}>
      <Header>Header</Header>
      <Content>Content</Content>
      <Footer>Footer</Footer>
    </Layout>
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
})