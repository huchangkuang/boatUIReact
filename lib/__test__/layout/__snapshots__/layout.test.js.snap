// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout should be div 1`] = `
<div
  className="boat-layout"
  style={
    {
      "height": 500,
      "width": 400,
    }
  }
>
  <div
    className="boat-layout-header"
  >
    Header
  </div>
  <div
    className="boat-layout-content"
  >
    Content
  </div>
  <div
    className="boat-layout-footer"
  >
    Footer
  </div>
</div>
`;
